src/components/sectioningConfirm/SectioningConfirm.tsx

```diff
@@ -77,8 +77,8 @@ export default function SectioningConfirm({
   useEffect(() => {
     const subscription = service.subscribe((state) => {
       if (state.matches('confirmed')) {
-        const sources = extractLabwareFromFlagged(sourceLabware);
```

Comment:
What is the difference between `sourceLabware` and `state.context.sourceLabware`?